index.js

```diff
@@ -233,6 +236,7 @@ export const Classes = {
 		content: 'content',
 		sequence: 'sequence',
 		sequencedActivity: 'sequenced-activity',
+		sequenceDescription: 'sequence-description',
```

Comment:
This one seems odd, too. Shouldn't there just be a `description` entity on a `sequence` entity?

Reply:
It looks like this one is used for modules, whereas `sequence-activity` is used for a content topic. In that sense, I think this is "this is a description-type sequence (i.e. module)" rather than "this is a description _of_ a sequence".

Reply:
Ah interesting, thanks for the explanation.